elastic/apm-server
# Makefile: fix golint target

## Motivation/summary

Fix `make golint` issue noted in https://github.com/elastic/apm-server/issues/3725

## Checklist

~- [ ] I have updated [CHANGELOG.asciidoc](https://github.com/elastic/apm-server/blob/master/CHANGELOG.asciidoc)~

I have considered changes for:
~- [ ] documentation~
~- [ ] logging (add log lines, choose appropriate log selector, etc.)~
~- [ ] metrics and monitoring (create issue for Kibana team to add metrics to visualizations, e.g. [Kibana#44001](https://github.com/elastic/kibana/issues/44001))~
~- [ ] automated tests (add tests for the code changes, all [**unit** tests](https://github.com/elastic/apm-server/blob/master/TESTING.md) pass locally)~
~- [ ] telemetry~
~- [ ] Elasticsearch Service (https://cloud.elastic.co)~
~- [ ] Elastic Cloud Enterprise (https://www.elastic.co/products/ece)~
~- [ ] Elastic Cloud on Kubernetes (https://www.elastic.co/elastic-cloud-kubernetes)~

## How to test these changes

1. Add the following to main.go:

```go

// Bar foos
func Foo() {
}
```

2. Run `make golint`
3. Check output is something like

```
/home/you/projects/apm-server/main.go:32:1: comment on exported function Foo should be of the form "Foo ..."
make: *** [Makefile:203: golint] Error 1
```

## Related issues

Closes https://github.com/elastic/apm-server/issues/3725
